### PR TITLE
Add portuguese translation 💪🇦🇴🇧🇷🇨🇻🇹🇱🇬🇶🇬🇼🇲🇴🇲🇿🇵🇹🇸🇹💪

### PR DESCRIPTION
**Pull request by ademar111190**
Hi, I just add this Portuguese translation on Breez fork so I'm sharing it here now.

About the translation, Brazilian Portuguese is my mother language, there are important differences in Brazilian Portuguese and other variants, especially on the conjugation of the second person (singular and plural), gerund x periphrastic, vocabulary, etc… So to have a _wide_ Portuguese support it is necessary at least the European and African variations, nevertheless, it is still the same language so all Portuguese speakers can understand it perfectly.

### CODE REVIEW

**Comment by amugofjava** [comment]
Thank you @ademar111190 that's fantastic.

**Comment by amugofjava** [comment]
Hi @ademar111190,

No, I'm not waiting for anything from you - just not had chance yet to merge this PR.

Thanks.

File: lib/l10n/intl_pt.arb [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+    "type": "text",
+    "placeholders": {}
+  },
+  "subscribe_button_label": "SEGUIR",
+  "@subscribe_button_label": {
+    "description": "Subscribe button label",
+    "type": "text",
+    "placeholders": {}
+  },
+  "unsubscribe_button_label": "NÃO SEGUIR",
```
Comment (bitcoinuser) [inline lib/l10n/intl_pt.arb:38]:
```suggestion
  "unsubscribe_button_label": "DEIXAR DE SEGUIR",
```

File: lib/l10n/intl_pt.arb [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+    "type": "text",
+    "placeholders": {}
+  },
+  "subscribe_label": "Seguir",
+  "@subscribe_label": {
+    "description": "Subscribe label",
+    "type": "text",
+    "placeholders": {}
+  },
+  "unsubscribe_label": "Não Seguir",
```
Comment (bitcoinuser) [inline lib/l10n/intl_pt.arb:62]:
```suggestion
  "unsubscribe_label": "Deixar de seguir",
```

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Minor changes

**Comment by amugofjava** [comment]
Thank you both @ademar111190 & @bitcoinuser for your contributions. I have not merged these yet as I am currently working on new features and will have new text strings. Would you be happy to translate these new strings when they become available?

**Pull request closed without merging**